frontend/src/libs/components/burger-menu/styles.module.scss

```diff
@@ -15,10 +15,12 @@
   color: $red-light;
 }
 
+$menu-offset: 8px;
+
 .menu {
   position: absolute;
-  top: 110%;
-  right: 0;
+  top: #{$menu-offset + 60px};
+  right: #{$menu-offset - 29px};
```

Comment:
better write result in px, don't like the calculation

Reply:
@kalinkaaaa14 this is pure serverside and shows where and why the values are came from + is more flexible.

On the other side, how can I turn down the please of our the only one beloved femine coach :)